src/Serilog.Sinks.Console/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.0-beta-*",
+  "version": "2.0.0-*",
```

Comment:
Ah, I missed that, will kill the build that I will have just kicked off... Nice catch.